packages/cactus-plugin-ledger-connector-cdl-socketio/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# TODO
+# Install connector as yarn package like in @hyperledger/cactus-plugin-ledger-connector-besu
+
+FROM node:12
```

Comment:
@outSH NodeJS v12 is End-Of-Life since 2020 and stopped receiving security patches as of last Spring. Unless there is a very strong reason to keep using it I highly recommend upgrading to v18 or v20

Source: https://endoflife.date/nodejs

Reply:
@petermetz Fixed

Reply:
@outSH Thank you!